File: Assignments/Assignment_17/p2.c
#include<stdio.h>
void main(){

        int row;
        printf("Enter the no of rows : \n");
        scanf("%d",&row);
        for(int i=1;i<=row;i++){

                int alp=96+i;
                for(int j=1;j<=i;j++){

                        printf("%c\t",alp);
                        alp--;
                }printf("\n\n");
        }
}

File: Flutter/Assignment/Square pattern set-2/p2.dart
import "dart:io";
void main(){
	print("Enter the no. of rows:");
	int? row=int.parse(stdin.readLineSync()!);
	for(int i=1;i<=row;i++){
		int x=row-i+1;
		for(int j=1;j<=row;j++){
			if(i>=2 && j<i){
				stdout.write("*\t");
			}else{
				stdout.write("${x}\t");
				x--;	
			}
		}
		print("");
	}
}
